test/acceptance-tests/workspace-devtime/step-definitions/reading-artifacts/listModels.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+var util = require('util');
+var async = require('async');
+var app = require('../../../../../');
+var loopback = require('loopback');
+var path = require('path');
+var ModelConfig = app.models.ModelConfig;
+var exampleWorkspace = path.resolve(__dirname, '../../../../../example');
+
+module.exports = function() {
+  var numberOfExpectedModels = 0;
+  var numberOfAvailableModels = 0;
+  var testcase = this;
+
+  this.Given(/^I have a workspace containing (\d+) models$/, function(numberOfExpectedModels, next) {
```

Comment:
the number of models is determined by the entries in model-config.json; the example project's model-config.json will give that number